### ISSUE TITLE
Document multiple databases support

### ISSUE THREAD

**Issue by pawelpacana** [RailsEventStore/rails_event_store#937]

**Comment by mzanini** [comment]
this. 
Was not able to make it work following instructions from[ this article](https://blog.arkency.com/rails-multiple-databases-support-in-rails-event-store/). 
Code:
```ruby
event_repository = RailsEventStoreActiveRecord::EventRepository.new(
    EventStoreBase, 
    serializer: RubyEventStore::NULL,
  )
```
Error:
<img width="751" alt="Screenshot 2023-06-01 at 1 58 20 PM" src="https://github.com/RailsEventStore/rails_event_store/assets/6310811/bbdec945-d17f-4c1a-8e45-c709f2e98da5">

[Looks like ](https://github.com/RailsEventStore/rails_event_store/blob/master/ruby_event_store-active_record/lib/ruby_event_store/active_record/event_repository.rb#L10)we now have to pass a `model_factory` to point to another AR base class?

**Comment by mpraglowski** [comment]
Yep, you are right. The behavior described in blog post has changed in commit https://github.com/RailsEventStore/rails_event_store/commit/8a6e7c30a9b3a76c5565ab2eb48481bfdf9e3b69 (and later other changes).

The valid code now is:

```ruby
  event_repository = RubyEventStore::ActiveRecord::EventRepository.new(
    model_factory: RubyEventStore::ActiveRecord::WithAbstractBaseClass.new(EventStoreBase), 
    serializer: RubyEventStore::NULL,
  )
```
